src/components/featureBanner/featureBanner.module.scss

```
@@ -0,0 +1,271 @@
+    text-decoration: none;
+    margin-top: $space-2;
+    transition: gap var(--duration-fast) var(--ease-out);
+
+    &:hover {
+      gap: $space-3;
+    }
+  }
+
+  // ── overlayLeft variant ──
+  &.overlayLeft {
+    height: pxToRem(380);
```

Stylelint (17.6.0)

[error] Expected "pxToRem" to be "pxtorem" (function-name-case)

(function-name-case)